client/assets/stylesheets/shared/mixins/_placeholder.scss

```diff
@@ -5,8 +5,7 @@
 
 @mixin placeholder( $color-property: --color-neutral-5 ) {
 	animation: loading-fade 1.6s ease-in-out infinite;
-	// this is just a solid color, but gives background-position and background-size capabilities ;)
-	background-image: linear-gradient( var( $color-property ), var( $color-property ) );
+	background: var( $color-property );
```

Comment:
I'm very concerned about making this change for Gutenboarding… do we know what kind of impact this has across Calypso? This is a very widely used mixin.

Reply:
I see, this was just added _for Gutenboarding_ here: https://github.com/Automattic/wp-calypso/pull/40915

Reply:
> This is a very widely used mixin.

Yes, keeping it simple brings back some peace of mind 🧘